azure-pipelines.yml

```diff
@@ -839,7 +841,7 @@ jobs:
       set -ex
 
       CLI_VERSION=`cat $SYSTEM_ARTIFACTSDIRECTORY/metadata/version`
-      RPM_NAME=$(find $SYSTEM_ARTIFACTSDIRECTORY/rpm/ -type f -name "azure-cli-$CLI_VERSION-1.cm2.*.rpm" -printf '%f\n')
+      RPM_NAME=$(find $SYSTEM_ARTIFACTSDIRECTORY/rpm/ -type f -name "azure-cli-$CLI_VERSION-1.*.rpm" -printf '%f\n')
```

Comment:
What will be the new package name?

Reply:
`azure-cli-2.62.0-1.azl3.x86_64.rpm`

Reply:
Can we add some example names to the comment, including both Mariner 2 and Azure Linux 3? This will increase the readability.